src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package model;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruit;
+    private final int quantity;
+
+    public FruitTransaction(String symbol, String fruit, int quantity) {
+        this.operation = convertStringIntoOperation(symbol);
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
The constructor of your model class should not contain any business logic. It's better to move the conversion of the string into an operation to a separate service or utility class. This will make your model class cleaner and easier to understand.